staff/lgimeneza/portfolio/styles/main.css

```diff
@@ -32,15 +51,65 @@ img{
 }
 
 .main-title{
-    padding-left: 30px;
+    padding-left: calc(100% / 2 - 400px);
     color: rgb(82, 81, 81);
     font-family: 'Roboto Condensed', sans-serif;
 }
+
+@media all and (max-width: 650px) {
+    .site-title{
+        font-size: 1.5em;
+    }
+}
+
+@media all and (max-width: 550px) {
+    .site-title{
+        font-size: 1em;
+    }
+}
+
 .section-header{
     padding-left: 10px;
     padding-top:5px;
     padding-bottom: 5px;
     color: white;
     background-color: rgba(54, 54, 54, 0.7);
     font-family: 'Montserrat', sans-serif;
-}
\ No newline at end of file
+}
+
+#skylab-ref{
+    color: white;
+}
+
+.site-footer{
+    padding-top:10px;
+    padding-bottom: 10px;
+    color: white;
+    background-color: rgba(54, 54, 54, 0.7);
+    text-align: center;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 13px;
+}
```

Comment:
Maybe you can group some common rules (check section-header and site-footer)

![](https://media.giphy.com/media/xk0vJLeQvzaes/giphy.gif)